cmudb/extensions/tscout/tscout_feature_gen.py

```diff
@@ -0,0 +1,160 @@
+import re
+import sys
+from pathlib import Path
+from typing import List, Tuple
+
+from clang.cindex import TypeKind
+
+# We're assuming that this script is housed in postgres/cmudb/extensions/tscout
+# We calculate the path of TScout relative to this extension and add it to the PythonPath temporarily.
+TSCOUT_EXTENSION_PATH = Path(__file__).parent
+CODEGEN_TEMPLATE_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, "operating_unit_codegen.c")
+CODEGEN_FILE_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, "operating_unit_features.h")
+
+TSCOUT_PATH = Path.joinpath(TSCOUT_EXTENSION_PATH, Path("../../tscout/"))
```

Comment:
I completely agree!
Didn't feel right to add this file to the same directory as Tscout, just to retain IDE support. 
Felt that this script (at this stage) was simple enough to get by, without the IDE.
Long term solution for this would be to restructure our modules, as you suggested.

Reply:
Yea, let's just leave this conversation open and we can come back to it in the future